browser/js/app-initializer.js

```diff
@@ -146,11 +148,15 @@ export class AppInitializer {
 					syndication.init(flags);
 				}
 
+				if (this.enabledFeatures.feedback) {
+					feedback.init();
+				}
+
 				if (this.enabledFeatures.roe) {
 					roe.init(flags);
 				}
 
-				if (this.enabledFeatures.evenMoreJanky) {
+				if (this.enabledFeatures.evenMoreJanky && flags.get('qualtrics')) {
```

Comment:
Is this the correct place for the flag?  I would have though it should be on line 151.

Reply:
Poop! Good spot